tests/managed-service/test_storageclassclaim.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import ManageTest, tier1, polarion_id
+from ocs_ci.ocs.resources.storageclassclaim import create_storageclassclaim
+from ocs_ci.helpers.helpers import create_ocs_object_from_kind_and_name
+
+log = logging.getLogger(__name__)
+
+
+@tier1
```

Comment:
I'm a bit confused: if storageclassclaim test is relevant for managed ocs only, I'd expect a marker 'skip if not managed service'. If it's more generic, I'd expect it not to be in managed_service folder.

Reply:
This is relevant for MS only. Added 'ms_consumer_required' marker.